pkg/infra/pulumi_aws/deploylib.ts

```diff
@@ -1003,174 +1008,6 @@ export class CloudCCLib {
             )
     }
 
-    public setupRDS(orm: string, args: Partial<aws.rds.InstanceArgs>) {
```

Comment:
IMO it would have been nice to make the changes and the move separately so it was easier to tell what changed